master/buildbot/status/build.py

```diff
@@ -629,6 +629,14 @@ def get_artifacts(self):
                 self.artifacts = artifacts
             return self.artifacts
 
+    def get_trigger_steps_time(self):
+        trigger = 0
+        trigger_steps = [step.finished - step.started for step in self.steps if step.finished and str(TriggerType) == step.step_type ]
```

Comment:
@mariangemarcano probably I understood you wrongly. I thought we should not include subclasses. That is why it doesn't work of course. Let me redo that.